packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -7,72 +7,145 @@
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
 import {
-  getColorV8,
   DEFAULT_THEME,
   retrieveComponentStyles,
   getLineHeight,
-  focusStyles
+  focusStyles,
+  getColor
 } from '@zendeskgarden/react-theming';
 
 import { IGlobalAlertProps } from '../../types';
 
-const COMPONENT_ID = 'notifications.global-alert';
+const COMPONENT_ID = 'notifications.global_alert';
 
 interface IStyledGlobalAlertProps {
-  alertType: IGlobalAlertProps['type'];
+  $alertType: IGlobalAlertProps['type'];
 }
 
+const lightDarkOptions = (lightOffset: number, darkOffset: number) => ({
+  light: { offset: lightOffset },
+  dark: { offset: darkOffset }
+});
+
 /**
  * 1. Shifting :focus-visible from LVHFA order to preserve `color` on hover
  */
-const colorStyles = (props: ThemeProps<DefaultTheme> & IStyledGlobalAlertProps) => {
+const colorStyles = ({ theme, $alertType }: ThemeProps<DefaultTheme> & IStyledGlobalAlertProps) => {
   let borderColor;
   let backgroundColor;
   let foregroundColor;
   let anchorHoverColor;
   let anchorActiveColor;
-  let focusColor;
-
-  switch (props.alertType) {
-    case 'success':
-      borderColor = getColorV8('successHue', 700, props.theme);
-      backgroundColor = getColorV8('successHue', 600, props.theme);
-      foregroundColor = getColorV8('successHue', 100, props.theme);
-      anchorHoverColor = props.theme.palette.white;
-      anchorActiveColor = props.theme.palette.white;
-      focusColor = 'successHue';
+  let focusVariable;
+
+  switch ($alertType) {
+    case 'success': {
+      borderColor = getColor({
+        variable: 'border.successEmphasis',
```

Comment:
Interested in thoughts around this use of emphasis vs non-emphasis. In general, I tried to use non-emphasis with foreground and emphasis with background. Does this align with design intent?

Reply:
Note this doesn't apply across all components, and at some points foreground is using emphasis (see StyledAlert). 

I'd like to ensure that when a consumer updates a color variable, it cascades through these components in the least confusing way. 

Also, let me know if I'm overthinking it. 😛